<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <width>1312</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>-5</y>
      <width>1321</width>
      <height>851</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="background.qrc">:/newPrefix/Users/fadli ahmad fahrezi/Pictures/bg3.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>70</y>
      <width>1311</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(167, 167, 167, 0); </string>
     <string notr="true">QScrollBar:vertical {
    border: none;
    background: rgb(183, 177, 242);
    width: 20px;
    margin: 0px 0px 0px 0px;
}

QScrollArea{
    border: none;
    background: #ffffff;
    width: 20px;
    margin: 0px 0px 0px 0px;
}</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-793</y>
       <width>1290</width>
       <height>2076</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar:vertical {
    border: none;
    background: rgb(183, 177, 242);
    width: 12px;
    margin: 0px 0px 0px 0px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QFrame" name="frame_10">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>680</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>200</y>
           <width>1101</width>
           <height>171</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aplikasi ini membantu anda memperbaiki kualitas foto lama&lt;/p&gt;&lt;p&gt;Cukup unggah foto lama anda dan nikmati hasil yang lebih tajam dan jelas&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_11">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>680</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="InputImage_4">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>30</y>
           <width>751</width>
           <height>461</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(167, 167, 167, 50);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="LoadButton_4">
         <property name="geometry">
          <rect>
           <x>563</x>
           <y>510</y>
           <width>140</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Load Image</string>
         </property>
        </widget>
        <widget class="QPushButton" name="GaussianButton_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>540</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(183, 177, 242);</string>
         </property>
         <property name="text">
          <string>Gaussian</string>
         </property>
        </widget>
        <widget class="QPushButton" name="EqualizationButton_4">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>540</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(183, 177, 242);</string>
         </property>
         <property name="text">
          <string>Equalization</string>
         </property>
        </widget>
        <widget class="QPushButton" name="GreyscaleButton_4">
        <widget class="QPushButton" name="BinerButton">
         <property name="geometry">
          <rect>
           <x>910</x>
           <y>510</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(183, 177, 242);</string>
         </property>
         <property name="text">
          <string>Biner</string>
         </property>
        </widget>
        <widget class="QPushButton" name="BinerButton_4">
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>610</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(183, 177, 242);</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_11">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>610</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(183, 177, 242);</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_12">
        <widget class="QPushButton" name="GrayscaleButton">
         <property name="geometry">
          <rect>
           <x>770</x>
           <y>510</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(183, 177, 242);</string>
         </property>
         <property name="text">
          <string>Grayscale</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_12">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>680</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="OutputImage_4">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>30</y>
           <width>751</width>
           <height>461</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(167, 167, 167, 50);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QSlider" name="BrightnessSlider_4">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>560</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="ContrastSlider_4">
         <property name="geometry">
          <rect>
           <x>59</x>
           <y>630</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="SharpeningSlider_4">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>560</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="SaturationSlider_4">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>630</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_23">
         <property name="geometry">
          <rect>
           <x>59</x>
           <y>529</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Brightness</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_24">
         <property name="geometry">
          <rect>
           <x>58</x>
           <y>599</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Contrast</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_25">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>528</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sharpening</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_26">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>600</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Saturation</string>
         </property>
        </widget>
        <widget class="QSlider" name="HueSlider_4">
         <property name="geometry">
          <rect>
           <x>550</x>
           <y>560</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="ValueSlider_4">
         <property name="geometry">
          <rect>
           <x>550</x>
           <y>630</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_27">
         <property name="geometry">
          <rect>
           <x>516</x>
           <y>538</y>
           <width>101</width>
           <x>549</x>
           <y>536</y>
           <width>68</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Hue</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_28">
         <property name="geometry">
          <rect>
           <x>517</x>
           <y>608</y>
           <width>111</width>
           <x>549</x>
           <y>606</y>
           <width>68</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
        <widget class="QPushButton" name="SaveButton_4">
         <property name="geometry">
          <rect>
           <x>860</x>
           <y>530</y>
           <width>251</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Save Image</string>
         </property>
        </widget>
        <widget class="QPushButton" name="SaveButtonEx">
         <property name="geometry">
          <rect>
           <x>860</x>
           <y>600</y>
           <width>251</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Save .exl</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ResetButton">
         <property name="geometry">
          <rect>
           <x>1070</x>
           <y>410</y>
           <width>141</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(183, 177, 242);</string>
         </property>
         <property name="text">
          <string>Reset Image</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_13">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>770</y>
      <width>1311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(167, 167, 167, 30);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_14">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(167, 167, 167, 30);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>601</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(167, 167, 167, 0);</string>
     </property>
     <property name="text">
      <string>Perbaikan Kualitas Citra pada Foto Lama</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1313</width>
     <width>1312</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
